biosimulations/apps/dispatch-service/src/app/app.controller.ts

```diff
@@ -52,20 +55,22 @@ export class AppController {
       return { message: 'Unsupported simulator was provided!' };
     }
 
-    const omexPath = data.filepathOnDataStore;
-    const sbatchName = `${uuid()}.sbatch`;
+    const sbatchName = `${data.simulationId}.sbatch`;
     const sbatchPath = path.join(sbatchStorage, sbatchName);
 
     this.logger.log('SBatch path: ' + sbatchPath);
 
     // Generate SBATCH script
+    // TODO: Rename singularity images biosimulations_ to biosimulators_ on HPC and build new images according to /simulator from DB
     const simulatorString = `biosimulations_${data.simulator}_${data.simulatorVersion}`;
-    const hpcTempDirPath = `${this.configService.get('hpc.hpcBaseDir')}/${data.uniqueFilename.split('.')[0]
-      }`;
+    const hpcTempDirPath = `${this.configService.get('hpc.hpcBaseDir')}/${data.simulationId}`;
     const sbatchString = this.sbatchService.generateSbatch(
       hpcTempDirPath,
       simulatorString,
-      data.filename
+      data.omexFileName,
+      // 'http://99d1dcb20ebd.ngrok.io', 
```

Comment:
Lets get this out of the codebase

Reply:
While testing in my local, the URL from libs appending `//api` instead `localhost:3333/api`. So, I'm serving it through `ngrok` to download the omex file on HPC(for local run only). We might need to change the local config in the URL file in libs.

Reply:
I'll remove